specification/ml/_types/Model.ts

```diff
@@ -64,6 +64,7 @@ export class ModelSizeStats {
   model_bytes: ByteSize
   model_bytes_exceeded?: ByteSize
   model_bytes_memory_limit?: ByteSize
+  output_memory_allocator_bytes?: long
```

Comment:
```suggestion
  output_memory_allocator_bytes?: ByteSize
```

Even if it's always a `long`, we can generate better code, if we consistently use `ByteSize` for sizes.